Makefile

```diff
@@ -137,8 +144,11 @@ helm-uninstall: ; $(info $(M)...Begin to helm-uninstall.)  @ ## Helm-uninstall.
 	kubectl delete -f https://raw.githubusercontent.com/kubesphere/ks-installer/master/roles/ks-core/prepare/files/ks-init/role-templates.yaml
 
 # Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: vet ;$(info $(M)...Begin to run tests.)  @ ## Run tests.
-	export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=2m; go test ./pkg/... ./cmd/... -covermode=atomic -coverprofile=coverage.txt
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
```

Comment:
I am thinking could we add a new entry, for example `dev-env` to prepare a local development environment, and download all necessary resources.

Reply:
What do you mean `all necessary resources`? All I know are `kubebuilder` related resources. The `dev-env` dir name looks good to me.

Reply:
I mean a new make entry,  like https://github.com/kubernetes/ingress-nginx/blob/main/Makefile#L151

Reply:
done